main.go

```diff
@@ -133,6 +133,12 @@ func main() {
 	if migrateDB {
 		os.Exit(0)
 	}
+	err = migration.UpdateUsersWithDefaultCluster(db, configuration.GetOpenShiftClientApiUrl())
```

Comment:
It would be better to use a migration sql file for this but not sure how we could pass the default cluster URL...

Reply:
Yeah, that's why added this here - because the default url is cluster-specific. Also, this code could be taken out later on since every user would be seeded with the default cluster api url henceforth.

Reply:
This is a part of data migration. We should not remove this migration step. We should version this change in DB. We do have access to configuration in our migration code where we run our scripts. It looks like we support custom arguments in our sql templates.

Reply:
Thanks, trying out this
`m = append(m, steps{ExecuteSQLFile("010-add-cluster-to-user.sql", "my cluster")})`